### DISCUSSION TITLE
Table wit all detection times in "Daily Detections" viw

### DISCUSSION THREAD

**Discussion by Smo-RBR** [Nachtzuster/BirdNET-Pi#29] [Ideas]
Can we change the tables in Overview and Daily charts to show all detections instead of the first and last ten detections?
This would be great.

**Comment by MueJosh** [comment]
Well... this topic is good. 

On the "original" BirdNET-Pi my chart looked like this: 
![image](https://github.com/Nachtzuster/BirdNET-Pi/assets/145150281/b3ba97ee-c006-44cc-a1be-3297fbb969c8)

Unfortunately the script doesn't work on this fork and I don't have the time right now to tinker with it. 
If you're interested I can attach it. 
The readings value automatically changed the size of the chart. It worked great :) 


You can also edit the daily_plot.py (`sudo nano /home/USERNAME/birdnet-pi/scripts/daily_plot.py`
Line 36 contains the number of readings - edit it to your desire. But don't expect that it will look like in the screeenshot...

**Comment by Nachtzuster** [comment]
Looking into it.
I got the code from @Svardsten53, so it's not too hard
